Challenges/madlibs_pt2.rb

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env ruby
+
+
+puts "--" *10
+puts "|Madlibs Blanks|"
+puts "--" *10
+
+blanks = ['adjective', 'plural Noun', 'noun', 'adjective', 'part of the body', 'adjective',' plural noun', 'part of the body', 'adjective', 'adverb']
+
+vowels = ['a', 'e'. 'i', 'o', 'u']
```

Comment:
You have a typo here. The period after the "e" should be a comma.

Reply:
⬆️ Before looking at the code, I tried running this locally and got an error from this. 

````
> ruby madlibs.rb

madlibs.rb:10: syntax error, unexpected tSTRING_BEG, expecting '('
vowels = ['a', 'e'. 'i', 'o', 'u']
````